scripts/pi-hole/php/header.php

```diff
@@ -424,145 +424,151 @@ function pidofFTL()
                 <!-- Home Page -->
                 <li<?php if($scriptname === "index.php"){ ?> class="active"<?php } ?>>
                     <a href="index.php">
-                        <i class="fa fa-home"></i> <span>Dashboard</span>
+                        <i class="fa fas fa-home"></i> <span>Dashboard</span>
```

Comment:
`fa` is an alias for `fas`, correct? Then only one is needed.

Reply:
I know the documentation says they are identical, but they are not.

`class="fa fas fa-list-ul"` (as expected):
![screenshot from 2019-01-07 18-54-02](https://user-images.githubusercontent.com/16748619/50784354-b8c3e980-12ad-11e9-9220-c9c1e3c22d02.png)

`class="fas fa-list-ul"` (justification/width lost):
![screenshot from 2019-01-07 18-54-18](https://user-images.githubusercontent.com/16748619/50784380-c6796f00-12ad-11e9-9495-87be5a1b04c2.png)

It took me a while to get the things back into shape after updating `font-awesome`...

Reply:
Is there a similar issue if you only use `fa`?

Reply:
This works fine.